go-controller/pkg/node/ovspinning/ovspinning_linux_test.go

```diff
@@ -191,7 +218,7 @@ func mockFeatureEnableFile(t *testing.T, data string) func() {
 	previousValue := featureEnablerFile
 	featureEnablerFile = f.Name()
 
-	os.WriteFile(featureEnablerFile, []byte(data), 0)
+	os.WriteFile(featureEnablerFile, []byte(data), 0777)
```

Comment:
Why 777?

Reply:
No specific reason. That flag is used only when the file doesn't exist and it is created by the WriteFile. Reverting to 0